integration-tests/features/steps/common.py

```diff
@@ -8,11 +8,16 @@
 from behave import given, then, when
 from urllib.parse import urljoin
 import jsonschema
-import requests
+import requests 
 
 import jwt
 from jwt.contrib.algorithms.pycrypto import RSAAlgorithm
 
+# Do not remove - kept for debugging
+# import logging
+# logging.basicConfig(level=logging.INFO)
+# log = logging.getLogger(__name__)
+
```

Comment:
Actually we should keep it here uncommented, the logging should work corectly in Behave if configured properly in environment.py